src/main/resources/db/migration/h2/V1.7__organisational-unit-children-data.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO `organisational_unit` (`id`, `parent_id`, `code`, `abbreviation`, `name`) VALUES
```

Comment:
NOTE - We said we weren't going to run migrations for h2 if you remember. just put in one file please

Reply:
The reason i havent put in one file is because i am going to delete later. Its much easier/cleaner to delete one file, than lines in a file.